services/backend/src/resources/user/http/self_information.go

```diff
@@ -0,0 +1,32 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/gofiber/fiber/v2"
+	httpModels "github.com/uvulpos/go-svelte/src/resources/user/http/http-models"
+)
+
+//	@Summary		Get own user permissions
+//	@Description	Returns all set permission for authorized users
+//
+//	@Tags			user
+//	@Produce		json
+//
+//	@Success		200	{object}	httpModels.SelfInformationModel
+//	@Failure		404	{string}	string	"The requested data could not be found."
+//	@Failure		500	{string}	string	"An error occurred while processing your request. Please try again later. "
+//
+//	@Router			/api/v1/self [get]
+func (s *UserHandler) GetSelfInformation(c *fiber.Ctx) error {
+	userID := c.Locals("user-uuid").(string)
+
+	selfInformation, selfInformationErr := s.service.GetUserSelfInformationByID(userID)
+	if selfInformationErr != nil {
+		status, _, message := selfInformationErr.HttpError()
+		return c.Status(status).SendString(message)
```

Comment:
no because this is only what the user gets and the user should not care about certain errors. However I have the possibility to display more information about the error, but this is all generated inside the HttpError Function and should not be part of the http handler